### PR TITLE
NIFI-9786 Add debug logs if keystore cannot be loaded

### PR DESCRIPTION
**Pull request by tamas-horvath**
Thank you for submitting a contribution to Apache NiFi.

Please provide a short description of the PR here:

#### Description of PR

_Enables logging for KeyStoreUtils isStoreValid validation failure; closed improvement NIFI-9786._

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically `main`)?

- [x] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [-] Have you written or updated unit tests to verify your changes?
- [x] Have you verified that the full build is successful on JDK 8?
- [x] Have you verified that the full build is successful on JDK 11?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [x] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [x] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [x] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/KeyStoreUtils.java [hunk 1]
```diff
@@ -401,6 +401,7 @@ public static boolean isStoreValid(final URL keystore, final KeystoreType keysto
             return true;
 
         } catch (Exception e) {
+            logger.debug("Keystore could not be loaded", e);
```
Comment (exceptionfactory) [inline nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/KeyStoreUtils.java:404]: Recommend including the keystore location and keystore type for additional debugging:
```suggestion
            logger.debug("Keystore [{}] Type [{}] load failed", keystore, keystoreType, e);
```

**Review by exceptionfactory — CHANGES_REQUESTED** [review]
Thanks for the improvement @tamas-horvath, this will be helpful for troubleshooting. I recommended an adjustment to include the keystore location and keystore type for additional detail in the log.

**Review by exceptionfactory — APPROVED** [review]
Thanks for the update and contribution @tamas-horvath, looks good! +1 merging.

**Comment by tamas-horvath** [comment]
Sample exception:
`2022-03-21 16:49:41,127 DEBUG [Validate Components Thread-2] o.a.nifi.security.util.KeyStoreUtils Keystore [file:/Users/tamashorvath/repos/cloudera-truststore.jks] Type [JKS] load failed
java.io.IOException: Keystore was tampered with, or password was incorrect
	at sun.security.provider.JavaKeyStore.engineLoad(JavaKeyStore.java:782)
	at sun.security.provider.JavaKeyStore$JKS.engineLoad(JavaKeyStore.java:56)
	at sun.security.provider.KeyStoreDelegator.engineLoad(KeyStoreDelegator.java:224)
	at sun.security.provider.JavaKeyStore$DualFormatJKS.engineLoad(JavaKeyStore.java:70)
	at java.security.KeyStore.load(KeyStore.java:1445)
	at org.apache.nifi.security.util.KeyStoreUtils.isStoreValid(KeyStoreUtils.java:399)
	at org.apache.nifi.ssl.StandardSSLContextService.validateTruststoreFile(StandardSSLContextService.java:495)
	at org.apache.nifi.ssl.StandardSSLContextService.validateTruststore(StandardSSLContextService.java:459)
	at org.apache.nifi.ssl.StandardSSLContextService.validateStore(StandardSSLContextService.java:376)
	at org.apache.nifi.ssl.StandardSSLContextService.customValidate(StandardSSLContextService.java:197)
	at org.apache.nifi.components.AbstractConfigurableComponent.validate(AbstractConfigurableComponent.java:130)
	at org.apache.nifi.controller.AbstractComponentNode.computeValidationErrors(AbstractComponentNode.java:741)
	at org.apache.nifi.controller.AbstractComponentNode.performValidation(AbstractComponentNode.java:708)
	at org.apache.nifi.controller.service.StandardControllerServiceNode.performValidation(StandardControllerServiceNode.java:500)
	at org.apache.nifi.controller.AbstractComponentNode.performValidation(AbstractComponentNode.java:722)
	at org.apache.nifi.components.validation.StandardValidationTrigger.trigger(StandardValidationTrigger.java:52)
	at org.apache.nifi.components.validation.TriggerValidationTask.run(TriggerValidationTask.java:42)
	at org.apache.nifi.engine.FlowEngine$2.run(FlowEngine.java:110)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.security.UnrecoverableKeyException: Password verification failed
	at sun.security.provider.JavaKeyStore.engineLoad(JavaKeyStore.java:780)
	... 24 common frames omitted`

**Pull request closed without merging**